bin/alluxio

```diff
@@ -34,6 +34,7 @@ function printUsage {
   echo -e "  killAll <WORD>\t Kill processes containing the WORD."
   echo -e "  copyDir <PATH>\t Copy the PATH to all worker nodes."
   echo -e "  clearCache    \t Clear OS buffer cache of the machine."
+  echo -e "  docGen        \t Generate configuration docs."
```

Comment:
Perhaps we should rename to `Gen*`. Also `docGen` is a bit vague, since this could be javadocs or the actual docs, both of which would not be correct.